src/main/java/babysitter/BabySitter.java

```diff
@@ -18,22 +19,19 @@ public BabySitter(int startTime, int endTime, int bedTime) {
 		this.bedTime = convertAm(bedTime);
 	}
 
-	boolean isValidWorkHours(int startTime, int endTime) {
+	boolean isValidWorkHours() {
 		return (startTime >= 17 && startTime < 28) && (endTime <= 28 && endTime > 17);
 	}
 
-	boolean invalidUserEntry(int startTime, int endTime, int bedTime) {
-		boolean invalidTracker = false;
-		int[] invalids = new int[] { 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 };
-		try (Scanner sc = new Scanner(System.in)) {
-			for (int i = 0; i < invalids.length; i++) {
-				if (invalids[i] == sc.nextInt()) {
-					invalidTracker = true;
-				}
-			}
-		}
-		return invalidTracker;
-
+	boolean isValidUserEntry() {
+		final int[] invalids = IntStream.rangeClosed(5, 16).toArray();
+		
+		boolean startBeforeEnd = startTime < endTime;
+		boolean startDuringValidHour = IntStream.of(invalids).noneMatch(x -> x == startTime);
+		boolean endDuringValidHour = IntStream.of(invalids).noneMatch(x -> x == endTime);
+		boolean bedTimeDuringValidHour = IntStream.of(invalids).noneMatch(x -> x == bedTime);
+		
+		return startBeforeEnd && startDuringValidHour && endDuringValidHour && bedTimeDuringValidHour;
```

Comment:
Instead of opening a new scanner (which doesn't get any more input, so it doesn't go anywhere), you can use your babysitter's already-defined instance variables to check validity instead.

Also, because the hours are already converted by the constructor method, you could simplify this method to one line:

```
boolean isValidUserEntry() {
  return startTime < endTime && startTime >= 17 && endTime >= 17 && bedTime >= 17;
}
```